### ISSUE TITLE
Scatter3D not remember it's visual state in certain conditions

### ISSUE THREAD

**Issue by kirsan31** [plotly/plotly.js#5560]
Visual state is zoom level, orientation, position, etc...
Checking for saving state: simply click on one of this control buttons:  ![image](https://user-images.githubusercontent.com/17767561/111898963-5bee1d00-8a3a-11eb-83b9-9c5b6d7f8a21.png) or call `Plotly.restyle`. None of these actions should change the visual state.

**Bugs briefly:**
1. Mouse wheel zooming not saving last zoom lvl (always reverted to previous zoom lvl).
2. Touch pad zooming/rotating/moving not saved at all.
---------------------------

Сonditions under which the visual state is not preserved:

**Zoom without mouse click - by mouse wheel.**
Each odd iteration of zoom level with mouse wheel - will not be remembered. Why odd? Because plotly will save previous zoomlvl after next zoom. In other words - mouse wheel zoomlvl will be reverted 1 step back after restyle or other _refresh_ actions.

https://user-images.githubusercontent.com/17767561/111899838-13396280-8a40-11eb-87ba-4849c2602679.mp4

This make mouse wheel zooming not usable with updating by time interval (`Plotly.restyle`) plots. Side note - if your click any where in the plot after mouse wheel zoom - zoomlvl will be stored.

**Any plot manipulation with touch pad.**
Here same behavior - touch any where in the plot after manipulating - will store visual state. See video below.  

https://user-images.githubusercontent.com/17767561/111899939-bc805880-8a40-11eb-9d72-be0d15e8e907.mp4

Repro: https://codepen.io/k_sze/pen/WXgNoJ (from #2181). You can click buttons or call `restyle()` function in console.

**Comment by nicolaskruchten** [comment]
Thanks for the bug report! I think it might be related to https://github.com/plotly/plotly.js/issues/5050 and/or https://github.com/plotly/plotly.js/issues/5005

**Comment by kirsan31** [comment]
@nicolaskruchten yes it's looks similar. But I need to point out that mouse wheel zoom bug exist in every Scatter3D plots without additional conditions (like `react` in your examples) :(

**Comment by nicolaskruchten** [comment]
Indeed, something is going wrong with our camera positioning system... No one on the Plotly team has available bandwidth to look into these right now but we'd love some help fixing these issues!
